src/main/java/org/spongepowered/api/entity/living/complex/EnderDragon.java

```diff
@@ -48,4 +47,11 @@
      */
     Optional<EnderCrystal> getHealingCrystal();
 
+    /**
+     * Gets the phase manager.
```

Comment:
Care to elaborate what the phase manager does? Remember, for us working in implementation, we know exactly what it does, but to plugin developers, it's completely foreign.